app/user_test.go

```diff
@@ -253,8 +251,8 @@ func TestUpdateOAuthUserAttrs(t *testing.T) {
 
 	var user, user2 *model.User
 	var gitlabUserObj oauthgitlab.GitLabUser
-	user, gitlabUserObj = createGitlabUser(t, th.App, username, email)
-	user2, _ = createGitlabUser(t, th.App, username2, email2)
+	user, gitlabUserObj = createGitlabUser(t, th.App, 1, username, email)
```

Comment:
Given that the error in the ticket shows that it had issues updating the same email and username, should we also remove the randomness here of the email/username that relies on the `NewId` that still uses `uuid.NewRandom()`?

Reply:
no, a UUID is random enough that we don't need to do that. If we can't rely on UUIDs not colliding, then Mattermost is completely broken anyway.